styles/alphav3-1.json

```diff
@@ -0,0 +1,286 @@
+				},
+				{
+					"slug": "quarternary",
+					"color": "var(--bcds-surface-color-border-active)",
+					"name": "Public Sector – Quarternary"
+				},
+				{
+					"slug": "quinary",
+					"color": "var(--bcds-theme-gold-80)",
+					"name": "BC Yellow – Quinary"
+				},
+				{
+					"slug": "senary",
+					"color": "var(--bcds-icons-color-danger)",
+					"name": "Danger – Senary"
+				},
+				{
+					"slug": "septenary",
+					"color": "var(--bcds-icons-color-success)",
+					"name": "Success – Septenary"
+				},
+				{
+					"slug": "octonary",
+					"color": "var(--bcds-icons-color-warning)",
+					"name": "Warning – Octonary"
+				},
+				{
+					"slug": "nonary",
+					"color": "var(--bcds-icons-color-primary)",
+					"name": "Icons primary – Nonary"
+				},
+				{
+					"slug": "denary",
+					"color": "var(--bcds-icons-color-secondary)",
+					"name": "Icons secondary – Denary"
+				},
+				{
+					"slug": "ondenary",
+					"color": "var(--bcds-icons-color-info)",
+					"name": "Icons info – Ondenary"
+				},
+				{
+					"slug": "duodenary",
+					"color": "var(--bcds-icons-color-disabled)",
+					"name": "Icons disabled – Duodenary"
+				},
+				{
+					"slug": "tridenary",
+					"color": "var(--bcds-surface-color-overlay-default)",
+					"name": "Dark overlay – Tridenary"
+				},
+				{
+					"slug": "quadenary",
+					"color": "var(--bcds-typography-color-secondary-invert)",
+					"name": "Inverted text (non-white) – Quadenary"
+				},
+				{
+					"slug": "gray-80",
+					"color": "var(--bcds-theme-gray-80)",
+					"name": "Gray 80"
+				},
+				{
+					"slug": "gray-30",
+					"color": "var(--bcds-theme-gray-30)",
+					"name": "Gray 30"
+				},
+				{
+					"slug": "gray-10",
+					"color": "var(--bcds-theme-gray-10)",
+					"name": "Gray 10"
+				},
+        {
+					"slug": "heading-line",
+					"color": "var(--bcds-theme-gold-90)",
+					"name": "Heading Line"
+				},
+				{
+					"slug": "foreground",
+					"color": "var(--bcds-typography-color-primary)",
+					"name": "Foreground"
+				},
+				{
+					"slug": "background",
+					"color": "var(--bcds-surface-color-background-white)",
+					"name": "Background"
+				},
+				{
+					"slug": "transparent",
+					"color": "transparent",
+					"name": "Transparent"
+				}
+			]
+		},
+		"spacing": {
+			"blockGap": null
+		},
+		"typography": {
+			"fontSizes": [
+				{
+					"slug": "small",
+					"size": "var(-—bcds-typography-font-size-label)",
+					"name": "Small"
+				},
+				{
+					"slug": "normal",
+					"size": "var(-—bcds-typography-font-size-body)",
+					"name": "Normal"
+				},
+				{
+					"slug": "large",
+					"size": "var(-—bcds-typography-font-size-large-body)",
+					"name": "Large"
+				},
+				{
+					"slug": "extra-large",
+					"size": "var(-—bcds-typography-font-size-h5)",
+					"name": "Extra large"
+				}
+			],
+			"fontFamilies": []
+		},
+		"custom": {}
+	},
+	"styles": {
+		"blocks": {
+			"core/list-item": {
+				"css": "& { font-size: 1rem }"
```

Comment:
Should these use `--bcds-typography-font-size-body` (this is also 1 rem).

Reply:
this is a thought, this PR is mostly creating the additional file, and not updating the values, although i have done some.